### PR TITLE
Update npm script

### PR DESCRIPTION
**Pull request by jonterrylee**
Update NPM script for start.

Some web platforms like Heroku or Azure don't support CROSS ENV.

### CODE REVIEW

**Comment by jonterrylee** [comment]
will need to change core to set default to production and development if specified

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "node": ">=8.6"
   },
   "scripts": {
-    "start": "cross-env NODE_ENV=production; node src/index.js;",
+    "start": "node src/index.js",
     "nodemon": "cross-env NODE_ENV=development; node scripts/nodemon;",
```
Comment (andrewmarcus) [inline package.json]: We don't need to remove it here as well?

**Comment by sureshHARDIYA** [comment]
Would be nice to have this merged?

**Comment by Robert-W** [comment]
@jonterrylee Is this good to go

**Pull request merged**